packages/botonic-plugin-contentful/src/contentful/queue.ts

```diff
@@ -53,4 +54,5 @@ export interface QueueFields extends ContentWithNameFields {
   queue: string;
   schedule?: contentful.Entry<ScheduleFields>;
   searchableBy?: contentful.Entry<SearchableByKeywordsFields>[];
+  indicationText: string;
 }
```

Comment:
always create or extend a unit test when you modify a library
You can extend packages/botonic-plugin-contentful/tests/contentful/contents/queue.test.ts